js/arrays/1-optional-parameter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+function returnEntries(array, upTo) {
```

Comment:
Subtler way to do this:
````
function returnEntries(array, upTo) {
    upTo = upTo || 1;
    return array.slice(0,2);
}
````
or even arguably:
````
function returnEntries(array, upTo) {
    return array.slice(0,upTo || 1);
}
````
Though the latter is too obtuse for me, I prefer the stabalisers on with the extra variable